opensuse-base-image/src/main/docker/scripts/check-create-pgdb-impl.sh

```diff
@@ -117,16 +166,36 @@ function check_variables {
   fi
 }
 
+function create_pgpass_file {
+  echo "INFO: Creating .pgpass file at $pgpassFile"
+
+  if echo "$database_host:$database_port:$database_name:$database_username:$database_password" > "$pgpassFile"; then
+    echo "INFO: Successfully wrote to .pgpass file"
+  else
+    echo "ERROR: Failed to write to .pgpass file"
+    exit 1
+  fi
+
+  if chmod 0600 "$pgpassFile"; then
+    echo "INFO: Successfully set permissions on .pgpass file"
+  else
+    echo "ERROR: Failed to set permissions on .pgpass file"
+    exit 1
+  fi
+
+  export PGPASSFILE="$pgpassFile"
+  echo "INFO: PGPASSFILE environment variable set to $PGPASSFILE"
+}
+
 function check_db_exist {
   echo "INFO: Checking database existence..."
 
 # Need to set password for run
 # Sending psql errors to file, using quiet grep to search for valid result
- if  PGPASSWORD="$database_password" \
-   PGAPPNAME="$database_appname" psql --username="$database_username" \
+ if PGAPPNAME="$database_appname" psql --username="$database_username" \
    --host="$database_host" \
    --port="$database_port" \
-   --variable database_name="$database_name" \
```

Comment:
`--variable database_name="$database_name"` no longer works when using .pgpass file

Reply:
I think just passing the db name directly, so:

```"$database_name"``` instead of  `--variable database_name="$database_name"`

Reply:
Did you try putting an equals sign after `--variable`?

```suggestion
   --variable=database_name="$database_name" \
```

Reply:
`--variable database_name="$database_name"` works again after using asterisks in the .pgpass file